### PR TITLE
Package melange-pancake.1.0

### PR DESCRIPTION
**Pull request by rolandpeelen**
### `melange-pancake.1.0`
Melange Pancake
A tiny lens library that works with Melange

---
* Homepage: https://github.com/rolandpeelen/melange-pancake
* Source repo: git+https://github.com/rolandpeelen/melange-pancake.git
* Bug tracker: https://github.com/rolandpeelen/melange-pancake/issues

---
:camel: Pull-request generated by opam-publish v2.3.0

### CODE REVIEW

**Comment by arbipher** [comment]
got from the opam-ci-lint

```
Your dune-project file indicates that this package requires at least dune 3.8 but your opam file only requires dune >= 1.0. Please check which requirement is the right one, and fix the other.
```

`melange` itself also [requires](https://github.com/melange-re/melange/blob/1a1f87a209f9677883563af24a99384219e07313/melange.opam#L10C3-L10C20) `"dune" {>= "3.8"}` so it should be fine to add this to your opam.

File: packages/melange-pancake/melange-pancake.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+synopsis: "Melange Pancake"
+description: "A tiny lens library that works with Melange"
+maintainer: ["Roland Peelen"]
+authors: ["Roland Peelen"]
+dev-repo: "git+https://github.com/rolandpeelen/melange-pancake.git"
+license: "MIT"
+homepage: "https://github.com/rolandpeelen/melange-pancake"
+bug-reports: "https://github.com/rolandpeelen/melange-pancake/issues"
+depends: [
+  "ocaml"
```
Comment (mseri) [inline packages/melange-pancake/melange-pancake.1.0/opam:11]:
```suggestion
  "ocaml" {>= "3.8"}
```

**Comment by mseri** [comment]
The tarball points at version 2.0.1 that defined pancake and not melange-pancake. The build fails immediately since dune does not know how to build this. I think you should make a new tag and release the version on your current master branch

**Comment by rolandpeelen** [comment]
Will fix this / push a new version! Thanks for the feedback! :)

**Pull request closed without merging**